code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -56,8 +61,48 @@
 						return
 					M.put_in_l_hand(src)
 			add_fingerprint(usr)
-			return
 
+//Proc for Contents transfer to other containers, bins, turf, and more possible additions.
+/obj/item/weapon/storage/proc/contentto(obj/dest_object, mob/user)
+	playsound(loc, "rustle", 50, 1, -5)
+	switch( storage_contents_dump_act(dest_object) )

```

Comment:
@Miauw62 @RemieRichards 

I'm three years into computer science stuff, and I've just been massively schooled - and reminded what OOP is. I'm one hell of a dolt! But I needed that, I guess. =p

Thanks for the advices.